Files/MultilingualResources/Files.fr-FR.xlf

```diff
@@ -2271,13 +2271,13 @@
         </trans-unit>
         <trans-unit id="StatusBarControlMultiselect.Text" translate="yes" xml:space="preserve">
           <source>Multiselect</source>
-          <target state="new">Multiselect</target>
+          <target state="new">Sélection Multiple</target>
```

Comment:
Change state to `translated` for translated strings.

Reply:
oh I see, I didn't know it... It's pretty useful to maintain translations

anyway, solved e5ac860